SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1332,6 +1335,55 @@ void USpatialPendingNetGame::SendJoin()
 	bSentJoinRequest = true;
 }
 
+Worker_EntityId USpatialNetDriver::SetupActorEntity(AActor* Actor)
```

Comment:
I disagree with these functions sitting in the NetDriver. Seems like a catch all. Could the SpatialReceiver or PackageMap be better places for them?

Reply:
This name is a little generic. How about `ReserverEntityIdForActor`?

Reply:
Not sure about using reserve in the function name as it could be misleading with regards to making entity ID reservations?

Reply:
`AllocateEntityIdForActor`